Python/bltinmodule.c

```diff
@@ -2198,6 +2198,11 @@ builtin_input_impl(PyObject *module, PyObject *prompt)
                 goto _readline_errors;
             assert(PyBytes_Check(po));
             promptstr = PyBytes_AS_STRING(po);
+            if ((Py_ssize_t)strlen(promptstr) != PyBytes_GET_SIZE(po)) {
+                    PyErr_SetString(PyExc_ValueError,
+                                    "input: prompt string cannot contain null characters");
+                    goto _readline_errors;
+            }
```

Comment:
```suggestion
            if ((Py_ssize_t)strlen(promptstr) != PyBytes_GET_SIZE(po)) {
                PyErr_SetString(PyExc_ValueError,
                                "input: prompt string cannot contain null "
                                "characters");
                goto _readline_errors;
            }
```

Reply:
@eryksun: I have edited your suggestion to keep "null characters" on the single line. It is convenient if you search for all "null character" in the code.